source/services/encryption/EncryptionService.ts

```diff
@@ -41,7 +42,16 @@ export async function encryptWithAesKey(user: User, text: string): Promise<strin
   return await Aes.encrypt(text, aesEncryptor.aesKey, aesEncryptor.initializationVector);
 }
 
-export async function decryptWithAesKey(user: User, cipher: string): Promise<string> {
+export async function encryptFormAnswers(user: UserInterface, forms: FormsInterface) {
+  const encryptedAnswers = await encryptWithAesKey(user, JSON.stringify(forms.answers));
+
+  forms.answers = { encryptedAnswers };
+  forms.encryption.type = 'privateAesKey';
+
+  return forms;
+}
```

Comment:
Heads up. Incoming forms data lack required properties encryption.type
See object updateCaseRequestBody in CaseActions.js on row 28 that's passed to encryptFormAnswers
Not sure if this is intendent?

Reply:
Thanks, will update request in a coming API PR. Current implementation can only be tested in storybook, story EncryptionService / Terminal demo: "Run symmetric key demo".